packages/docs/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "zos-docs",
-  "version": "2.3.0-alpha.1",
+  "version": "2.3.0-rc.3",
```

```suggestion
  "version": "2.3.0",
```
we should also run a `lerna bootstrap` or a local `npm install` inside the `docs` package to update the `package-lock.json` Would you like to include it in this PR?

---

I'll update manually for now, and look for a more robust solution later.